tex/alg-geom/localization.tex

```diff
@@ -78,6 +78,13 @@ \section{Spoilers}
 localizations will give us a way to more or less
 describe the sheaf $\OO_X$ completely.
 
+In other words,
+\begin{moral}
+	Localization is the purely algebraic way to \emph{define} the
+	ring of regular functions on a smaller open set from the ring of ``global''
+	regular functions.
+\end{moral}
```

Comment:
Somehow, this important idea eluded me for quite a long time... not sure if it's because of the "silly" green box above.

Not sure how much another green box can help drawing attention, maybe it's really inevitable that the reader would need to come to Spec construction to appreciate the value of localization.